tests/e2e/Adapter/Scopes/Relationships/OneToManyTests.php

```
@@ -2676,4 +2676,191 @@ public function testPartialUpdateWithStringIdsVsDocuments(): void
+
+        $database->createRelationship(
+            collection: 'author',
+            relatedCollection: 'article',
+            type: Database::RELATION_ONE_TO_MANY,
+            twoWay: true,
+            id: 'articles',
+            twoWayKey: 'author'
+        );
+
+        // Create some articles
+        $article1 = $database->createDocument('article', new Document([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$article1'. (undefined)

(UnusedLocalVariable)